soc/arm/nuvoton_npcx/npcx7/Kconfig.defconfig.series

```diff
@@ -41,6 +41,12 @@ config PWM_NPCX
 	help
 	  Enable support for NPCX PWM driver.
 
+config ADC_NPCX
+	default y
+	depends on ADC
+	help
+	  Enable support for NPCX ADC driver.
```

Comment:
This should briefly describe what features are supported

Reply:
Done.